Models/SSBTaxAssessment/GraphQLTypes/TaxAssessmentDatasetType.cs

```diff
@@ -0,0 +1,65 @@
+                    return (from municipality in municipalities
+                        let municipalityYears = context.Source.Value
+                            .Skip(NorwayTools.MunicipalityCodeToIndexTaxes[municipality] * 294)
+                            .Take(municipalityEntrySize)
+                            .ToList()
+                        select new LabeledValueTax
+                        {
+                            Municipality = NorwayTools.MunicipalityCodeToMunicipalityNameTaxes[municipality],
+                            TaxesForYear = new Collection<TaxesForAGivenYear>(years.Select(year =>
+                                    new TaxesForAGivenYear
+                                    {
+                                        Year = year,
+                                        Brutto = municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize],
+                                        LonnInnt = municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 1],
+                                        Uskstt = municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 2],
+                                        AllmennInnt =
+                                            municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 3],
+                                        BankInn = municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 4],
+                                        BrFormue = municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 5],
+                                        Gjeld = municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 6],
+                                        MedianBtoInnt =
+                                            municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 7],
+                                        MedianLonnInnt =
+                                            municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 8],
+                                        MedianUtlignSkatt =
+                                            municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 9],
+                                        MedianAlmInnt =
+                                            municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 10],
+                                        MedianBankInns =
+                                            municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 11],
+                                        MedianBtoFormue =
+                                            municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 12],
+                                        MedianGjeld =
+                                            municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize + 13]
+                                    })
+                                .ToList())
```

Comment:
Strange indentation

Reply:
That's where it goes when you write the code. Code Cleanup doesn't alter it either

Reply:
Huh. Interesting 🤔